src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfig.cs

```diff
@@ -1,14 +1,46 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
 {
     public class RpcWorkerConfig
     {
+        internal static readonly TimeSpan DefaultInitializationTimeout = TimeSpan.FromSeconds(10);
+
         public RpcWorkerDescription Description { get; set; }
 
         public WorkerProcessArguments Arguments { get; set; }
 
         public WorkerProcessCountOptions CountOptions { get; set; }
+
+        /// <summary>
+        /// Gets or sets the process startup timeout. This is the time from when the process is
+        /// launched to when the StartStream message is received.
+        /// </summary>
+        public TimeSpan ProcessStartupTimeout { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
new properties should be added to `WorkerProcessCountOptions` --> Please see PR: https://github.com/Azure/azure-functions-host/pull/6806 for more details on implementation.

This needs to be renamed to `WorkerProcessOptions` --> this was specifically added to be able to configure various values related to process start up